tools/appliance/systemvmtemplate/scripts/install_systemvm_packages.sh

```diff
@@ -68,9 +68,10 @@ function install_packages() {
     python-flask \
     haproxy \
     radvd \
-    sharutils genisoimage aria2 \
+    sharutils genisoimage \
```

Comment:
This may break one of the register template features that uses aria2, @nvazquez can you comment?

Reply:
I believe the use of aria2 was scrapped.

Reply:
aria2 Used by KVM hosts with "bypass Sec.Stor." during template download.

Just tested Add template (from remote http) - agent/java download the template... 
"Upload from local" uses POST, so should not be relevant.